docs/en/ufs/Aliyun-OSS.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: global
+title: Aliyun Object Storage Service
+---
+
+
+This guide describes how to configure [Aliyun OSS](https://intl.aliyun.com/product/oss){:target="_blank"} as Alluxio's under storage system. 
```

Comment:
what's this `{:target="_blank"}` part i see in lots of places?

Reply:
i added it so that external docs open in a new tab to keep alluxio traffic within alluxio docs. love it hate it?

Reply:
like the idea. if we should adopt this for all external links, we should have an automatic way to append this to all the external links rather than remembering to type it out for each one. maybe @Zijian-Zhu can help to find a way for this?